### DISCUSSION TITLE
Blackhole just stopped working

### DISCUSSION THREAD

**Discussion by Mrsupersadow** [ExistentialAudio/BlackHole#612] [Q&A]
Black hole has just stopped working recording my desktop audio can someone help me I haven't changed anything but I think it might be something to do with big surr version 11.2.3 im on a m1 mac mini if that helps

**Comment by devinroth** [comment]
That’s not enough information to help you.

Devin Roth

**Comment by nandoripa** [comment]
> Black hole has just stopped working recording my desktop audio can someone help me I haven't changed anything but I think it might be something to do with big surr version 11.2.3 im on a m1 mac mini if that helps

Try reinstalling Blackhole. I just reinstalled it and it works again for me. After updating Macos the system uninstalled it.
